robocopkaka/house-renting-app
# Ft setup circle and coverage

#### What does this PR do?
Adds CircleCI config and coverage using coveralls
#### Description of Task to be completed?
Adds CircleCI config and coverage using coveralls
#### How should this be manually tested?
* Pull develop into your branch and push a commit
* Check build on CircleCI and see if it passes
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories? (if applicable)
[#171429465](https://www.pivotaltracker.com/story/show/171429465)
